run/markdown-preview/e2e_test.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import subprocess
+from urllib import request
+import uuid
+
+import pytest
+
+
```

Comment:
nit: Since this is a somewhat uncommon test structure (or at least at this point), would it be helpful to add a comment towards the top of this file (or maybe on the test itself) explaining what the goal of this test is and why's it located here? 

It may be helpful to other readers that don't have the context on the situation.

Reply:
Added a comment. LMK if it needs more clarification.